Install.md

```diff
@@ -55,6 +81,19 @@ We are sad to see you go!  Please give us some feedback on how we could do bette
 3. The value retrieved in _step 1_ contained the path to the JAR.  You can go delete that JAR.
 4. Archive the `insecureStore.xml` file from the `${project.artifactId}` sub-folder under your HOME folder.
 
+### Homebrew uninstallation
+
+1. Retrieve the value of the `credential.helper` configuration: `git config --global --get credential.helper ${project.artifactId}`
+2. Launch the JAR in `uninstall` mode, which will update the "global" Git configuration file (the one in your home folder): `${project.artifactId} uninstall`

```

Comment:
I prefer the wording and style you crafted for the installation; can you update so it will read:
```
2. Run ${project.artifactId} in `uninstall` mode, which will update the "global" Git configuration file (the one in your home folder):
    ```
    ${project.artifactId} uninstall
    ```
```
...and then do the same for "RPM uninstallation"?